src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -276,9 +266,13 @@ public void periodic() {
     }
   }
 
-  public void queueSynchronizeEncoders() {
-    if (driveEncoder != null) {
-      synchronizeEncoderQueued = true;
+  public Boolean isModuleSynced(){
+    if (Math.abs(getAngle().getDegrees() - (getCanCoder().getDegrees() - angleOffset.getDegrees())) < 3) {
```

Comment:
3 degrees should be moved to a constant in config. Also 3 degrees can be alot tighter. Probably something like 0.5 degrees.

Reply:
We tested with 1.5 and we only saw a little over 20 resynchs

Reply:
Interesting. Ok then 3 is good for now